docs/tutorials/pantheon.md

```diff
@@ -55,7 +56,7 @@ Once you've started up your Pantheon site you will need to pull in your database
 
 ### 1. Using `lando pull`
 
-Lando provides a command for Pantheon sites called `lando pull` to get your database and files. **If you do not specify `--code`, `--database` or `--files` then `lando` will use the environment associated with your currently checked out `git branch`.**
+Lando provides a command for Pantheon sites called `lando pull` to get your database and files. **If you do not specify `--code`, `--database` or `--files` then `lando` will use the environment associated with your currently checked out `git branch`.** Please note, `lando pull` **does not** clone repos, it essentially does a `git pull` (merge) for convenience (see #2 above).
```

Comment:
**does not** clone repos, it ~~essentially~~ does a `git pull` (merge)

@achton I think we should eliminate the word *essentially*

Reply:
Yes, it is superfluous.